src/screens/Home/styles.ts

```diff
@@ -1,4 +1,5 @@
 import styled from 'styled-components/native';
+import Text from '../../components/Text';
```

Comment:
```suggestion
import Text from '../../components/Text';
```
Remove unused imports